MachineImages/Templates/Debian_i386/Template.json

```diff
@@ -0,0 +1,41 @@
+{
+    "variables": {
+        "name"             : "debian-i386",
```

Comment:
Must better to take 64bit version `debian-amd64` `http://cdimage.debian.org/debian-cd/8.2.0/amd64/iso-cd/debian-8.2.0-amd64-netinst.iso`